### DISCUSSION TITLE
Edge AI meeting

### DISCUSSION THREAD

**Discussion by mattjbr123** [NERC-CEH/rse_group#36] [General]
Here are my unedited handwritten notes, apologies I didn't have time to write them up neatly. Hope they are somewhat useful, and others can add their notes too
[edge_ai.pdf](https://github.com/user-attachments/files/16562112/edge_ai.pdf)
![Scratch - page 10](https://github.com/user-attachments/assets/97c49061-2ba4-4389-ac56-0b35d4643ed4)

**Comment by metazool** [comment]
Thank you! "Adaptive Monitoring" as a keyphrase caught my eye.

https://www.webpages.uidaho.edu/~beth/REM357_readings_files/adaptive%20monitoring.pdf - older paper on Adaptive Monitoring, no visible date but the references are all 1990s/2000s, not a bad thing! Very relevant to the topic of "are the research questions we are asking with our long-term monitoring projects still useful questions" within the NC-UK Integrated Environmental Monitoring theme...  #28 

For me this dovetails neatly with the question of _assurance_ for bioacoustic sensors - practical example being the [next-gen AMI trap](https://github.com/AMI-system/) with speculative [BirdPi](https://github.com/pddpauw/BirdPi) on it - how do you verify that you're only recording and analysing certain frequencies, or prove on request that you _haven't_ recorded anything heard before a trigger event? One would _hope_ the phone-based voice assistant people struggle with these questions all the time :D
